ocrd_monitor/docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       - ${MONITOR_DATA}:/data
       - ${MANAGER_KEY}:/id_rsa
       - shared:/run/lock/ocrd.jobs
+      - /home/weigelt/Work/OCR-D/kitodo_production_ocrd/_modules/ocrd_manager/ocrd_monitor/flask/app:/usr/local/ocrd-monitor/app
```

Comment:
Wait, what happened here?

Reply:
Sorry that was a volume mount for local dev purposes.